src/core/module/image/ImageService.ts

```diff
@@ -71,7 +72,7 @@ export class ImageService {
 
   async handleGetImagesOfUser(userId: number): Promise<ImageResponseJson[]> {
     try {
-      const images = await this.imageRepository.getByUserId(userId);
+      const images = await this.imageRepository.getByUserId(userId, null);
```

Comment:
async getByUserId(user_id: number, visibility: boolean):
cái hàm nó như này rõ type boolean chỗ này truyền null ???

Reply:
ok để chỉnh về default là null
null -> get cả public và private cho owner
còn true false thì get theo visibility

Reply:
đã restrict về type boolean mà đi để null chỗ này m chỉ cần thêm 1 param là is_ignore_visibility: boolean = false là oke

async getByUserId(
    user_id: number,
    visibility: boolean = true,
    is_ignore_visibility: boolean = false,
  ): Promise<Image[]> {
    if (is_ignore_visibility) {
      return this.database.query.images.findMany({
        where: (image, { eq }) => eq(image.userId, user_id),
      });
    }

    return this.database.query.images.findMany({
      where: (image, { eq }) => eq(image.userId, user_id) && eq(image.visibility, visibility),
    });
  } 

fix nó thành như này